crates/anstyle-termcolor/src/lib.rs

```diff
@@ -22,18 +22,10 @@ pub fn to_termcolor_spec(style: anstyle::Style) -> termcolor::ColorSpec {
     let mut style = termcolor::ColorSpec::new();
     style.set_fg(fg);
     style.set_bg(bg);
-    if effects.contains(anstyle::Effects::BOLD) {
-        style.bold();
-    }
```

Comment:
... thats weird.  `bold` has the `must_use` attribute.  Even if I add it to the type, its still not triggering.

I did a search and I don't think this bug is in any of the other libs